L1Trigger/L1THGCal/src/backend_emulator/HGCalHistoClusteringImpl_SA.cc

```diff
@@ -45,6 +45,7 @@ void HGCalHistoClusteringImplSA::runAlgorithm(HGCalTriggerCellSAPtrCollections&
   HGCalClusterSAPtrCollection protoClusters = triggerCellToCluster( clusteredTCs );
   HGCalClusterSAPtrCollection clusterAccumulation;
   clusterSum( protoClusters, readoutFlags, clusterAccumulation, clusterSums );
+  ClusterProperties(clusterSums);
```

Comment:
Rename to `clusterProperties` for consistency.

Reply:
done in 342389f